linux_os/guide/services/ssh/ssh_server/sshd_enable_x11_forwarding/rule.yml

```diff
@@ -22,7 +22,8 @@ identifiers:
     cce@rhel8: 82421-9
 
 references:
-    cis: 5.2.4
+    cis@rhel7: 5.2.4
```

Comment:
This reference is about disabling x11 forwarding, but the rule is about enabling x11 forwarding. Same for rhel8.

Reply:
Removed as well.